1.Practicals_C/Array_String_Structue/Strings/times_of_repeated_word_in_string.c

```diff
@@ -0,0 +1,34 @@
+#include<stdio.h>
+void main()
```

Comment:
Add #include<string.h> header file

Reply:
I have done the required changes.Thank you for suggesting...